x-pack/plugins/enterprise_search/public/applications/app_search/components/analytics/components/analytics_header.scss

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.analyticsHeader {
+  flex-wrap: wrap;
+
+  &__filters.euiPageHeaderSection {
+    width: 100%;
+    margin: $euiSizeM 0;
+  }
+}
```

Comment:
Worth adding a note or something about why this custom CSS is applied (since we largely try to avoid it?)

Reply:
I'm a little on the fence about it, we haven't added any comments so far on our CSS or previously had a history of doing so, so I'd probably want to consider it from a whole-app POV and discuss the benefits of doing so. I think so far the understanding is custom CSS consists mostly of EUI/layout overrides and hopefully it's easy to see what the CSS actually accomplishes by unchecking/hiding the custom CSS in a live browser